apps/console/src/features/validation/pages/validation-config-edit.tsx

```diff
@@ -0,0 +1,769 @@
+                    }));
+
+                    return;
+                }
+                dispatch(addAlert({
+                    description: t(
+                        "console:manage.features.validation.notifications.genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("console:manage.features.validation.notifications.genericError.message")
+                }));
+            })
+            .finally(() => {
+                setSubmitting(false);
+            });
+    };
+
+    return (
+        <PageLayout
+            pageTitle={ t("console:manage.features.validation.pageTitle") }
+            title={ (
+                <>
+                    { t("console:manage.features.validation.pageTitle") }
+                    <Label size="medium" className="preview-label ml-2">
+                        { t("common:preview") }
+                    </Label>
+                </>
+            ) }
+            description={ (
+                <>
+                    { t("console:manage.features.validation.description") }
+                </>
+            ) }
+            data-componentid={ `${ componentId }-page-layout` }
+            backButton={ {
+                "data-testid": `${ componentId }-page-back-button`,
+                onClick: handleBackButtonClick,
+                text: isApplicationRedirect ?
+                    t("console:manage.features.validation.goBackToApplication")
+                    : t("console:manage.features.validation.goBackToValidationConfig")
+            } }
+            bottomMargin={ false }
+            contentTopMargin={ true }
+            pageHeaderMaxWidth={ true }
+        >
+            <Ref innerRef={ pageContextRef }>
+                <Grid
+                    className="mt-3"
+                >
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column width={ 16 }>
+                            <EmphasizedSegment className="form-wrapper" padded={ "very" }>
+                                <div className="validation-configurations password-validation-configurations">
+                                    { (!isValidationLoading && !isLoading)  ?
+                                        (<Form
+                                            id={ FORM_ID }
+                                            initialValues={ initialFormValues }
+                                            uncontrolledForm={ true }
+                                            validate={ null }
+                                            onSubmit={
+                                                (values: ValidationFormInterface) => handleUpdateMyAccountData(values)
+                                            }
+                                            enableReInitialize={ true }
+                                        >
+                                            { /*<Field.Radio*/ }
```

Comment:
Lets remove the commented code if we are not planning on immediately bring this in. Add a comment in the PR with the commit that has this change.